src/thor/worker.cc

```diff
@@ -96,6 +96,16 @@ thor_worker_t::thor_worker_t(const boost::property_tree::ptree& config,
 thor_worker_t::~thor_worker_t() {
 }
 
+std::string serialize_to_pbf(Api& request) {
+  std::string buf;
```

Comment:
It's likely a bit unoptimized to allocate the string every time but it should still be  very little work in comparison to actually running pathfinding or mapmatching or similar.